src/topTripCardList/TopTripCardList.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { cloneElement, createRef } from 'react'
+import cc from 'classcat'
+
+import { TripCardProps } from 'tripCard/TripCard'
+
+interface TopTripCardListProps {
+  readonly children: React.ReactElement<TripCardProps>[]
+  readonly className?: Classcat.Class
+  readonly width?: string
+  readonly isWrapped?: boolean
+}
+
+const ref = createRef<HTMLDivElement>()
+
+const TopTripCardList = ({ children, className = '', isWrapped = false }: TopTripCardListProps) => {
+  const tripCards = React.Children.toArray(children).filter(tripCard =>
+    React.isValidElement(tripCard),
+  )
+  return (
+    <div className={cc(['kirk-topTripCardList-wrapper', className])} ref={ref}>
```

Comment:
Why do you need the ref?

Reply:
We don't, removed

Reply:
I think I added it before handling the project to Jérôme, I had in mind to retrieve DOM info from it.
Can be removed now.